ui/public/locales/en.json

```diff
@@ -1197,7 +1197,7 @@
 "label.maxinstance": "Max instances",
 "label.maxiops": "Max IOPS",
 "label.maxmembers": "Max members",
-"label.maxmemory": "Max. memory (MiB)",
+"label.maxmemory": "Max. memory (MB)",
```

Comment:
are we sure it is not in multitudes of 1024*1024 instead of 1.000.000?

Reply:
MiB is appropriate here; our memory calculations are not dividing bytes by 10^6. Is there a specific place you're seeing something not expected?

Reply:
```suggestion
"label.maxmemory": "Max. memory (MiB)",
```